R/makeTreeSummarizedExperimentFromBiom.R

```diff
@@ -224,3 +244,77 @@ makeTreeSummarizedExperimentFromBiom <- function(obj, ...){
     }
     return(colname)    
 }
+
+# Detect and clean non wanted characters from Taxonomy data if needed.
+.detect_taxa_artifacts_and_clean <- function(x, patterns) {
+    
+    # No cleaning
+    if (is.null(patterns)) {
+        warning("No Taxonomy cleaning was done to the data.",
+                call. = FALSE)
```

Comment:
Is it necessary to throw a warning on this - I think no extra cleaning should be the safe default, assuming that the biom file follows the standards? I would not throw warning when the file is simply being read as expected. Warnings are warranted if extra cleaning is done automatically.